package-overrides/github/angular/bower-material@0.9.0-rc1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "main": "index",
+  "format":"cjs",
+  "registry": "jspm",
+  "dependencies": {
+    "angular": "^1.3.15",
+    "angular-animate": "^1.3.15",
+    "angular-aria": "^1.3.15",
+    "css": "^0.1.0"
```

Comment:
These should all include the full dependency name like `"angular": "github:angular/bower-angular@^1.3.15"`.